lib/internal/util.js

```diff
@@ -179,6 +175,15 @@ function deprecate(fn, msg, code, useEmitSync) {
   return deprecated;
 }
 
+function deprecateInstantation(target, code, ...args) {
+  if (code !== undefined)
+    lazyValidateString()(code, 'code');
```

Comment:
Using `validateString` is pure overhead for internal functions
```suggestion
  assert(typeof code === 'string');
```

Reply:
I would keep the assert for consistency, the other functions validate the input